packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/create-transform-job.ts

```diff
@@ -59,6 +59,13 @@ export interface SageMakerCreateTransformJobProps extends sfn.TaskStateBaseProps
    */
   readonly modelName: string;
 
+  /**
+   * Configures the timeout and maximum number of retries for processing a transform job invocation.
+   *
+   * @default
```

Comment:
No default value is specified on the [API page](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_CreateTransformJob.html#sagemaker-CreateTransformJob-request-ModelClientConfig) or the [description page](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_ModelClientConfig.html) for the ModelClientConfig.

I have observed that the default value for `InvocationsMaxRetries` is 3 in practice, and `InvocationsTimeoutInSeconds` is less than 1200 (not sure about the exact value), but could be wrong.

Reply:
I'd say that's the value we should document as the default text - if it's not documented we should deploy it and find out :)